packages/editor/src/pages/Editor/store/samples/selectors.ts

```diff
@@ -1,8 +1,10 @@
 import { IState } from '../reducer';
 
+import { selectors as host } from 'script-lab-core/lib/modules/host';
+
 export const getMetadataByGroup = (state: IState): ISampleMetadataByGroup =>
   Object.values(state.samples || {})
-    .filter(sample => sample.host === state.host)
+    .filter(sample => sample.host === host.get(state))
```

Comment:
By not using a selector here, things broke when state shape changed. Now using a selector.